stl/inc/yvals_core.h

```diff
@@ -194,35 +193,39 @@
 // P1023R0 constexpr For std::array Comparisons
 // P1024R3 Enhancing span Usability
 // P1032R1 Miscellaneous constexpr
+// P1035R7 Input Range Adaptors
 // P1065R2 constexpr INVOKE
 //     (except the std::invoke function which is implemented in C++17)
 // P1085R2 Removing span Comparisons
 // P1115R3 erase()/erase_if() Return size_type
 // P1123R0 Atomic Compare-And-Exchange With Padding Bits For atomic_ref
 // P1135R6 The C++20 Synchronization Library
 // P1207R4 Movability Of Single-Pass Iterators
-//     (partially implemented)
 // P1208R6 <source_location>
 // P1209R0 erase_if(), erase()
 // P1227R2 Signed std::ssize(), Unsigned span::size()
 // P1243R4 Rangify New Algorithms
-//     (partially implemented)
 // P1248R1 Fixing Relations
+// P1252R2 Ranges Design Cleanup
 // P1357R1 is_bounded_array, is_unbounded_array
 // P1391R4 Range Constructor For string_view
 // P1394R4 Range Constructor For span
 // P1423R3 char8_t Backward Compatibility Remediation
 // P1456R1 Move-Only Views
 // P1474R1 Helpful Pointers For contiguous_iterator
+// P1522R1 Iterator Difference Type And Integer Overflow
+//     (technically conforming, but it would be nice to implement a 65-bit integer-like type)

```

Comment:
Should we file an `enhancement` issue for this?

Reply:
It's already listed in my tracking comment (https://github.com/microsoft/STL/issues/39#issuecomment-665333818) so yes, I'll file an issue before this PR merges.